cloud-formation/subscriptions-app.cf.json

```diff
@@ -205,7 +232,8 @@
                 "ConnectionDrainingPolicy": {
                     "Enabled" : "true",
                     "Timeout" : "60"
-                }
+                },
+                "CrossZone": "true"
```

Comment:
As we're about to do a CloudFormation change, I've drive-by'd this very small tweak (which we use in many other places) to enable cross-zone routing from the ELB, which is recommended in the Trusted Advisor.